### ISSUE TITLE
Installing go-ethereum on Mac OS X

### ISSUE THREAD

**Issue by JustinDrake** [ethereum/go-ethereum#582]
I've followed to the letter the instructions [here](https://github.com/ethereum/go-ethereum/wiki/Building-Instructions-for-Mac) (Option 1: Homebrew tap):

```
brew tap ethereum/ethereum
brew install ethereum
```

Now when I run the command `ethereum`, I get the following error:

```
dyld: Library not loaded: /usr/local/opt/readline/lib/libreadline.6.dylib
  Referenced from: /usr/local/bin/ethereum
  Reason: image not found
Trace/BPT trap: 5
```

What's up?

(OS X Yosemite 10.10.2, MacBook Air early 2015)

**Comment by caktux** [comment]
Try `brew install readline` for now, fixed in ethereum/homebrew-ethereum@74840ec

**Comment by JustinDrake** [comment]
Ok, that seems to have fixed it for me. Thanks.
